playwright-test/onboarding/inception_1.spec.js

```diff
@@ -2,37 +2,47 @@ const { test } = require('../fixtures');
 const { expect } = require('@playwright/test');
 const Tesseract = require('tesseract.js');
 
+// 1. Go to www.browserstack.com and log in with your demo account
+// 2. Start a Live session, as a part of the automation script
+// 3. Within the Live session, perform a Google search for “BrowserStack”
+
+// 1. Go to www.browserstack.com
 test.beforeEach(async ({ page }) => {
   await page.goto('https://www.browserstack.com/');
 });
 
 test.describe('Browserstack Inception Nicholas onboarding', () => {
 
   test('Browserstack Inception Omnibus POC', async ({ page }) => {
+    // log in with your demo account
     await page.waitForSelector('text=Sign in',{state:'visible'});
     await page.locator('text=Sign in').first().click();
     await page.waitForSelector('#user_email_login')
-    await page.fill('#user_email_login', process.env.BROWSERSTACK_DEMO_EMAIL);
-    await page.fill('#user_password', process.env.BROWSERSTACK_DEMO_PASSWORD);
+    await page.fill('#user_email_login', "nicholas+demo@browserstack.com" );
+    await page.fill('#user_password', process.env.BROWSERSTACK_DEMO_PASSWORD );
     await page.locator('#user_submit').click();
+    // Start a Live session, as a part of the automation script
     await expect(page).toHaveURL("https://live.browserstack.com/dashboard");
+    // Location and click Windows 11  
     await page.locator('[aria-label="Windows 11"]').click();
```

Comment:
This line assume that the page you land on upon login is the Live dashboard. In my case, I landed on the Automate dashboard. Consider a conditional statement that verifies you are on the Live dashboard. 

See session: https://automate.browserstack.com/dashboard/v2/builds/75a6469182acd32599565325e63215234e21f222/sessions/81121765abd0ccec98cad29b4c5bf46997e75172?user_id=4242492

Reply:
I removed the check and now its just a goto

```
    await page.goto('https://live.browserstack.com/dashboard');
```